demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -614,12 +646,20 @@ def is_outputs_for_reputations_commands_valid(self) -> bool:
         for command in commands:
             command_name = command.get("name")
             # look for reputations commands
-            if command_name in BANG_COMMAND_NAMES:
+            if (
+                command_name in BANG_COMMAND_NAMES
+                or command_name in MANDATORY_REPUTATION_CONTEXT_NAMES
+            ):
                 context_outputs_paths = set()
                 context_outputs_descriptions = set()
                 for output in command.get("outputs", []):
                     context_outputs_paths.add(output.get("contextPath"))
                     context_outputs_descriptions.add(output.get("description"))
+                    output_for_reputation_valid = (
+                        self.validate_reputation_name_spelling(
+                            command_name, output.get("contextPath")
+                        )
+                    )
```

Comment:
```suggestion
                    context_path = output.get("contextPath")
                    context_outputs_paths.add(context_path)
                    context_outputs_descriptions.add(output.get("description"))
                    output_for_reputation_valid = (
                        self.validate_reputation_name_spelling(
                            command_name, context_path
                        )
                    )
```
extract `context_path` into a variable